### ISSUE TITLE
Problem with lists of strings begin converted to unicode (AntsJointFusion).

### ISSUE THREAD

**Issue by hjmjohnson** [nipy/nipype#1863]
### Summary

A quick followup with specific details:
 
In AntsJointFusion interface:
 
    target_image = InputMultiPath(File(exists=True), argstr='-tg %s...',
                                  mandatory=True, desc='Target image(s)')
 
 
is generating a command line of: 

### Actual behavior
-t [u'/Shared/nopoulos/structural/DM1_MR/BAWEXPERIMENT_20170302/20170302_DM1_base_CACHE/singleSession_985_65222115/TissueClassify/BABC/t1_average_BRAINSABC.nii.gz']
 

### Expected behavior
Which generates an error.
 
What we want is a command line of:
-t ['/Shared/nopoulos/structural/DM1_MR/BAWEXPERIMENT_20170302/20170302_DM1_base_CACHE/singleSession_985_65222115/TissueClassify/BABC/t1_average_BRAINSABC.nii.gz']
 

### How to replicate the behavior
Newer version of python or nipype (not determined).

### Platform details:
please paste the output of: `python -c "import nipype; print(nipype.get_info()); print(nipype.__version__)"`

**Comment by hjmjohnson** [comment]
@satra 

The following is the report.rst file that is created during the Running of AntsJointFusion.

[report_cleaned.txt](https://github.com/nipy/nipype/files/821459/report_cleaned.txt)

I noticed that the inputs are all coded as unicode for the inputs.  I confirmed this by running the following: 'find . -name "report.rst" -exec grep "u'" {} \;' in the nipype cache directory.  Previous environments (other version of anaconda/nipype) did not exhibit this behavior.

I am running "conda 4.3.13" as my python environment, and the master branch of nipype from today (with my patch fixes).

The previous implementation relied on the fact that the command line for ants was compatible with the string representation of a list of strings. The current implementation builds the command line for ants.

The problem is that print the list of strings is how causing an extra 'u' character to be prefixed on the path file names that are embedded in lists.

I will remove the u prefix in the testing framework shortly.
